flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -209,9 +209,18 @@ public static StructLikeSet expectedRowSet(Table table, Record... records) {
   }
 
   public static StructLikeSet actualRowSet(Table table, String... columns) throws IOException {
+    table.refresh();
```

Comment:
the method below already calls `table.refresh()`. seems redundant here.

Reply:
This `table.refresh()` here is to get the latest snapshot id, so that the follow `actualRowSet` could read the records from the latest snapshot.

Reply:
got it. I just found it a little weird that we have to call `table.refresh` twice in this case.  I think your intention is to support `actualRowSet` for either latest snapshot or specific snapshotId. maybe define the overloaded method as below. Then this method can just pass in null for the `snapshotId`
```
actualRowSet(Table table, @Nullable Long snapshotId, String... columns) 

```

Reply:
Yeah, I like you idea !